compile.sh

```diff
@@ -5,3 +5,4 @@ RUSTFLAGS="-Zlocation-detail=none" cargo +nightly build --manifest-path rust/Car
 kotlinc -include-runtime kotlin/code.kt -d kotlin/code.jar
 #kotlinc-native -include-runtime kotlin/code.kt -d kotlin/code
 dart compile exe dart/code.dart -o dart/code --target-os=macos
+cd inko && inko build --opt=aggressive code.inko -o code && cd ..
```

Comment:
Inko creates a `build` directory for temporary files in the current working directory. Using `cd` here makes it a little easier to remove that in the `clean.sh` script.